models/public/cocosnet/model.yml

```diff
@@ -0,0 +1,170 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  Cross-domain correspondence network is a exemplar-based image translation model,
+  consisting of correspondence and translation parts. Model was pre-trained on ADE20k
+  dataset. For details see paper <https://arxiv.org/pdf/2004.05571> and repository
+  <https://github.com/microsoft/CoCosNet>.
+task_type: image_translation
+files:
+  - name: util/__init__.py
```

Comment:
Please place all source files in a subdirectory to avoid clutter in the download directory.